src/chromatic_adaptation.rs

```diff
@@ -0,0 +1,228 @@
+        let s_wp = Swp::get_xyz();
+        let d_wp = Dwp::get_xyz();
+        let adapt = self.get_cone_response();
+
+        let resp_src = multiply_xyz(&adapt.ma, &s_wp);
+        let resp_dst = multiply_xyz(&adapt.ma, &d_wp);
+        let z = T::zero();
+        let resp = [resp_dst[0]/resp_src[0] , z ,z ,z , resp_dst[1]/resp_src[1], z , z, z, resp_dst[2]/ resp_src[2]];
+
+        let tmp = multiply_3x3(&resp, &adapt.ma);
+        multiply_3x3(&adapt.inv_ma, &tmp)
+    }
+}
+
+
+impl<T, Swp, Dwp> TransformMatrix<T, Swp, Dwp> for  Method
+    where T: Float,
+          Swp: WhitePoint<T>,
+          Dwp: WhitePoint<T>
+{
+    fn get_cone_response(&self) -> ConeResponseMatrices<T> {
+        match *self {
+             Method::Bradford => {
+                ConeResponseMatrices::<T> {
+                    ma: [flt(0.8951000), flt(0.2664000), flt(-0.1614000),
+                         flt(-0.7502000), flt(1.7135000), flt(0.0367000),
+                         flt(0.0389000), flt(-0.0685000), flt(1.0296000)
+                         ],
+                    inv_ma: [flt(0.9869929), flt(-0.1470543), flt(0.1599627),
+                             flt(0.4323053), flt(0.5183603), flt(0.0492912),
+                             flt(-0.0085287), flt(0.0400428), flt(0.9684867)
+                            ],
+                }
+            }
+             Method::VonKries => {
+                ConeResponseMatrices::<T> {
+                    ma: [flt(0.4002400), flt(0.7076000), flt(-0.0808100),
+                         flt(-0.2263000), flt(1.1653200), flt(0.0457000),
+                         flt(0.0000000), flt(0.0000000), flt(0.9182200)
+                         ],
+                    inv_ma: [flt(1.8599364), flt(-1.1293816), flt(0.2198974),
+                             flt(0.3611914), flt(0.6388125), flt(-0.0000064),
+                             flt(0.0000000), flt(0.0000000), flt(1.0890636)
+                             ],
+                }
+            }
+             Method::XyzScaling => {
+                ConeResponseMatrices::<T> {
+                    ma: [flt(1.0000000), flt(0.0000000), flt(0.0000000),
+                         flt(0.0000000), flt(1.0000000), flt(0.0000000),
+                         flt(0.0000000), flt(0.0000000), flt(1.0000000)
+                         ],
+                    inv_ma: [flt(1.0000000), flt(0.0000000), flt(0.0000000),
+                             flt(0.0000000), flt(1.0000000), flt(0.0000000),
+                             flt(0.0000000), flt(0.0000000), flt(1.0000000)
+                             ],
+                }
+            }
+        }
+    }
+}
+
+
+///Converts color from one reference white point to another (Swp -> Dwp)
+pub trait ChromaticAdaptation<T, Swp, Dwp, Color>
+    where T: Float,
+          Swp: WhitePoint<T>,
+          Dwp: WhitePoint<T>
+{
+    fn chromatic_adaptation(&self, method: TransformMatrix<T, Swp, Dwp>, Swp, Dwp) -> Color;
```

Comment:
> Is it necessary to pass Swp and Dwp as arguments here, or does this also break down otherwise?

I was thinking that this will be easier that using traits
```
let lab: Lab<D50, T> = Lab::with_wp(0.5, 0.5 ,0.5)

let yxy = lab.chromatic_adaptation(D65, D50);
```
but can also be done as 
```
let yxy: Yxy<D50, _> = lab.chromatic_adaptation();
```
I'll change it.

> It would also be nice with a short form of this, that defaults to Bradford (I think that's generally considered the standard).

That's a good idea